android/build.gradle

```diff
@@ -1,4 +1,12 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
+apply plugin: 'com.android.library'
+
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+def isNewArchitectureEnabled() {
+    return project.hasProperty("newArchEnabled") && project.newArchEnabled == "true"
+}
 
 buildscript {
```

Comment:
Does it compile if you remove the complete buildscript block?

Reply:
Will have to try, did you notice there is now only one gradle file?

Reply:
Should have mentioned, this approach builds for me in both cases.

If Expo does not work, try in the `packages/expo-app` to run `yarn build -- --clean` and afterwards try again `yarn android`.